src/shared/utils/exportData/parseResponseValue.ts

```diff
@@ -30,6 +30,13 @@ import {
 import { joinWihComma } from '../joinWihComma';
 import { getAnswerValue } from '../getAnswerValue';
 
+const getTimeRangeValue = (
+  data: DecryptedDateRangeAnswer['value']['from'] | DecryptedDateRangeAnswer['value']['to'],
+  hasFallback = false,
+) =>
+  `hr ${hasFallback ? data?.hour ?? 0 : data?.hour}, min ${
```

Comment:
could you please move these conditions to variables to improve code readability a bit?

```
  const hour = hasFallback ? data?.hour ?? 0 : data?.hour;
  const minute = hasFallback ? data?.minute ?? 0 : data?.minute;
  
  return `hr ${hour}, min ${minute}`;
```